File: app/src/main/java/com/runtimepermission/MainActivity.java
package com.runtimepermission;

import android.os.Bundle;

public class MainActivity extends RunTimePermission {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        requestPermission(new String[]{android.Manifest.permission.ACCESS_FINE_LOCATION, android.Manifest.permission.ACCESS_COARSE_LOCATION}, new PermissionCallback() {
            @Override
            public void onGranted() {

            }

            @Override
            public void onDenied() {

            }
        });
    }
}

File: app/src/main/java/com/runtimepermission/MainActivity.kt
package com.runtimepermission

import android.os.Bundle


class MainActivity : RunTimePermission() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        requestPermission(arrayOf(android.Manifest.permission.ACCESS_FINE_LOCATION,
                android.Manifest.permission.ACCESS_COARSE_LOCATION)) { isGranted ->
            if (isGranted) {
                println("Permission granted")
            } else {
                println("Permission denied")
            }
        }
    }
}
